GameServerCore/Packets/Interfaces/IPacketNotifier.cs

```diff
@@ -495,19 +463,14 @@ public interface IPacketNotifier
         /// </summary>
         /// <param name="seconds">Amount of time till the pause ends.</param>
         /// <param name="showWindow">Whether or not to show a pause window.</param>
-        void NotifyPauseGame(int seconds, bool showWindow);
+        void NotifyPausePacket(Tuple<uint, ClientInfo> player, int seconds, bool isTournament);
```

Comment:
Rather than passing in the tuple, simply pass in the ClientInfo. Do this for all packet functions which were changed to use a Tuple.